boards/arm/stm32f746g_disco/stm32f746g_disco.dts

```diff
@@ -119,3 +120,36 @@
 		     &eth_txd0_pg13
 		     &eth_txd1_pg14>;
 };
+
+&quadspi {
+	pinctrl-0 = <&quadspi_clk_pb2 &quadspi_bk1_ncs_pb6
+		     &quadspi_bk1_io0_pd11 &quadspi_bk1_io1_pd12
+		     &quadspi_bk1_io2_pe2 &quadspi_bk1_io3_pd13>;
+	status = "okay";
+
+	n25q128a1: qspi-nor-flash@0 {
+		compatible = "st,stm32-qspi-nor";
+		label = "N25Q128A1";
+		reg = <0>;
+		qspi-max-frequency = <108000000>;
```

Comment:
This value does not wok on my end, but 107000000 is fine.
Maybe there is some tolerance to be taken into account between parts ?

Reply:
Setting it to 107MHz results in a qspi frequency of 72MHz. I chose to change the value to 100MHz, which also ends up at 72MHz.

Reply:
Ok, maybe it would be better to set 72MHz directly then (Less surprise for users and avoid useless computation).

Reply:
Ok, I can do that.

But what do you recommend I do for 769i_disco? I have set it to 70MHz because that is the most conservative spec from the data sheet. If we assume 70MHz should be used then `qspi-max-frequency` should be set to 54MHz to follow your reasoning.

But the flash can go significantly faster if more dummy cycles are allowed. Based on your knowledge of the qspi driver implementation, can you recommend a max frequency? Data sheet is here:
 https://www.mxic.com.tw/Lists/Datasheet/Attachments/7881/MX25L51245G,%203V,%20512Mb,%20v1.7.pdf

Reply:
I don't have strong opinions other than the max tested ok value